UDPServer/SiceUDPServer.cs

```diff
@@ -32,10 +36,11 @@ public void Connect()
             {
                 client = new UdpClient(ConnectionPort);
                 currentPort = ConnectionPort;
-                this.ConnectionStatus = true;
                 source = new CancellationTokenSource();
```

Comment:
There are two standards for referencing private variables
using underscore: _privateVar = assignedValue;
or this: this.privateVar = assignedValue;

While not important in the PoC, please keep it consistent.

Reply:
Removed. It wasn't a useful variable. Forgot to delete it before